### PR TITLE
update to the throttled serverless limitations

### PR DESCRIPTION
**Pull request by zuowei593**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by disksing — APPROVED** [review]

**Review by baiyuqing — APPROVED** [review]

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ For the first five TiDB Serverless clusters in your organization, whether they a
 - Row-based storage: 5 GiB
 - [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit): 50 million RUs per month
 
-Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will not be affected. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.
+Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will not be affected, but will be throttled automatically. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.
```
Comment (shizn) [inline tidb-cloud/manage-serverless-spend-limit.md:23]: Connections that were established prior to reaching the quota will remain active. However, they will be throttled to 10 RU/s.

Comment (shizn) [inline tidb-cloud/manage-serverless-spend-limit.md:23]: Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will remain active but will experience throttling. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.

**Review by shizn — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**